src/sass/components/_sidenav.scss

```diff
@@ -109,7 +109,7 @@ $level-styles: (
   // Indent the link text
   #{$indent-selector} > .rvt-sidenav__item-wrapper > .rvt-sidenav__link,
   #{$indent-selector} > .rvt-sidenav__link {
-    text-indent: $indent + px;
+    padding-left: $indent + px;
```

Comment:
Since we use `rem`s for units everywhere else we should probably convert these pixel values to rems as well. Not critical for this PR, but maybe at least add a note/comment to come back and fix that.

Reply:
I went ahead and updated the values from `px` to `rem` since it was a small amount of work to fix it. Let me know what you think about the changes!